scrum/tools/sprint_math.py

```diff
@@ -326,13 +361,13 @@ def calc_planned_next_sprint(self):
         pto_adjustment = (total_days_to_be_missed -
                           total_days_missed_last) / n_members
 
-        self.sp_next_sprint = math.ceil((self.sp_planned_completed + self.sp_unplanned_completed -
+        return math.ceil((self.sp_planned_completed + self.sp_unplanned_completed -
                                 avg_unplanned) / length_adjustment - pto_adjustment)
 
 # Call the calc function to get what next Sprint's estimate number of points should be
-calc_obj = SprintMath(sp_unplanned_total, sp_unplanned_remaining, sp_unplanned_partial_completed,
-                        total_done_list, sp_planned_partial_completed, sp_retro_completed,
-                        sp_retro_leftover, unplanned_past_sprints)
+calc_obj = SprintMath(SP_UNPLANNED_TOTAL, SP_UNPLANNED_REMAINING, SP_UNPLANNED_PARTIAL_COMPLETED,
```

Comment:
why are you making some variable capitalized?

Reply:
This is because pylint would not pass otherwise if I had not done this. Pylint sees those variables as global variables, thus the capitalization suggestion